elected_office/models.py

```diff
@@ -0,0 +1,651 @@
+# elected_office/models.py
+# Brought to you by We Vote. Be good.
+# -*- coding: UTF-8 -*-
+
+from django.db import models
+from django.db.models import Q
+from exception.models import handle_exception, handle_record_found_more_than_one_exception
+from wevote_settings.models import fetch_site_unique_id_prefix, fetch_next_we_vote_id_elected_office_integer
+import wevote_functions.admin
+from wevote_functions.functions import convert_to_int, extract_state_from_ocd_division_id, positive_value_exists
+
+
+logger = wevote_functions.admin.get_logger(__name__)
+
+
+class ElectedOffice(models.Model):
+    # The we_vote_id identifier is unique across all We Vote sites, and allows us to share our data with other
+    # organizations
+    # It starts with "wv" then we add on a database specific identifier like "3v" (WeVoteSetting.site_unique_id_prefix)
+    # then the string "electedoff", and then a sequential integer like "123".
+    # We keep the last value in WeVoteSetting.we_vote_id_last_contest_office_integer
+    we_vote_id = models.CharField(
+        verbose_name="we vote permanent id for this elected office", max_length=255, default=None, null=True,
+        blank=True, unique=True)
+    # The name of the elected office.
+    elected_office_name = models.CharField(verbose_name="name of the elected office", max_length=255, null=False,
+                                           blank=False)
+    elected_office_name_es = models.CharField(verbose_name="name of the elected office in Spanish", max_length=255,
+                                              null=True, blank=True, default=None)
+    # The offices' name as passed over by Google Civic. We save this so we can match to this office even
+    # if we edit the office's name locally.
+    google_civic_elected_office_name = models.CharField(
+        verbose_name="elected_office name exactly as received from google civic", max_length=255, null=True, blank=True)
+    # The unique ID of the election containing this contest. (Provided by Google Civic)
+    google_civic_election_id = models.CharField(verbose_name="google civic election id",
+                                                max_length=255, null=False, blank=False)
+    google_civic_election_id_new = models.PositiveIntegerField(
+        verbose_name="google civic election id", default=0, null=False, blank=False)
+    ocd_division_id = models.CharField(verbose_name="ocd division id", max_length=255, null=True, blank=True)
```

Comment:
ElectedOffice entries won't have a google_civic_election_id since they are permanent offices. We will want to use google_civic_elected_office_name to capture the "offices" > "name" given to us by https://developers.google.com/civic-information/docs/v2/representatives/representativeInfoByAddress